pom.xml

```diff
@@ -69,7 +69,7 @@
         <dependency>
             <groupId>com.contrastsecurity</groupId>
             <artifactId>contrast-sdk-java</artifactId>
-            <version>2.1.1</version>
+            <version>2.2-SNAPSHOT</version>
```

Comment:
I believe this should be just 2.2 
https://search.maven.org/#artifactdetails%7Ccom.contrastsecurity%7Ccontrast-sdk-java%7C2.2%7Cjar

Using it with 2.2-SNAPSHOT fails to find the resource, when I switch it to 2.2 it works fine. 
<img width="882" alt="screen shot 2017-01-27 at 10 14 14 am" src="https://cloud.githubusercontent.com/assets/15878949/22375803/8c7c98ea-e479-11e6-8f78-40895f3743bf.png">